aero.minova.core.application.system.service/doc/adoc/security-code.adoc

```diff
@@ -3,6 +3,40 @@
 In diesem Dokument wird beschrieben, wie speziell die Methode,
 welche die Sicherheitsmechanismen implementieren, funktionieren.
 
+== Temporäre Tabellen im Code
```

Comment:
Bin jetzt gerade verwirrt, warum diese Datei im Master ist (obwohl das Ticket 139 ja nicht geschlossen ist).
Auf jeden Fall denke ich, dass diese Doku als Javadoc in den Java-Quellcode gehört,
da es Implementierungs spezifisch ist.

Nur allgemeine Konzepte oder die API sollte als adoc dokumentiert werden.